### ISSUE TITLE
AWS v18.0.0 release

### ISSUE THREAD

**Issue by alex-dabija** [giantswarm/roadmap#846]
ℹ️ Team Phoenix is drafting AWS v18.0.0 with Kubernetes v1.23 support.
Target release date: 2022-08-31.
Release PR: ...

## Scope
- [x] https://github.com/giantswarm/roadmap/issues/1137;
- [x] https://github.com/giantswarm/roadmap/issues/1102;
- [x] https://github.com/giantswarm/roadmap/issues/1138;
- [x] [external cloud manager](https://github.com/giantswarm/roadmap/issues/1211)
- [x] Component upgrades.

## Pre-Release
- [x] Create release ticket on [Releases board](https://github.com/orgs/giantswarm/projects/136)
- [x] Make an announcement on #sig-product that team is doing a release. Include a link to the ticket from the release board.
- [x] Make sure release was tested by creating new cluster, upgrading from older version and with HelloWorld app.
- [x] Aggregate release notes from all changed components and add to `README.md` in the release directory.
- [x] Send draft release announcement to #sig-product for review ASAP.
- [x] Verify documentation is up to date.

## Releasing
- [x] Create release PR
[Instructions](https://intranet.giantswarm.io/docs/dev-and-releng/releases/)
    - [x] Define new release in `release.yaml` file.
    - [x] Activate new release by setting `state` to `active`.
    - [x] Deactivate old release if it's within same major version (`state: deprecated`). Verify with PO if you have doubts.
    - [x] Check versions of all components included. They shouldn't be lower than those in the previous release unless there was a rollback.

[Example PR](https://github.com/giantswarm/releases/pull/318)
- [x] Release relevant operators.
    - [x] Tag release branch.
    - [x] Bump operator version in draft release.
- [x] Make sure that WIP releases in releases repo branch are removed and not merged to master since the master branch is deployed automatically after merging PR and that would make WIP releases visible in customers’ installations which should generally not happen.

## Post-Release
- [x] Update front-end cluster with the new release.
- [x] Verify release is available on all installations. You can use [this script](https://github.com/giantswarm/azure-operator/blob/25bf788946c5405532b3b4b76c54b8e685ee291b/scripts/list-releases.sh).
- [x] Send out release announcement to all customers according to [these instructions](https://intranet.giantswarm.io/docs/dev-and-releng/releases/#release-notes).
- [x] PO subscribes to release posts on Slack to get notifications if there are follow-up messages.
- [x] Close release task on Releases board.

**Comment by njuettner** [comment]
`aws-cloud-controller-manager` needs to run as a static pod for now because of `aws-cni`. As soon as we can migrate to `cilium` we can run it as an app.

Background what happens if we deploy `aws-cloud-controller-manager` as an app:

- kubelet sets additional taint `[node.cloudprovider.kubernetes.io/uninitialized`](http://node.cloudprovider.kubernetes.io/uninitialized%60)  on nodes when using external cloud provider
- aws-cloud-controller would remove it after it verifies node are ready

- calico-crd-installer cannot run because the taint is not tolerated, quickfix would be tolerating the taint BUT
- chart-operator can also not initialized, even if we also allow chart-operator to tolerate the taint BECAUSE

- aws-node is erroring that it cannot find the right AWS zone to assign the right IP of the the correct subnet, it’s done by the label [failure-domain.beta.kubernetes.io/zone](http://failure-domain.beta.kubernetes.io/zone) which is on the nodes which is set by the aws-cloud-controller ```"error while retrieving eniconfig: ENIConfig.crd.k8s.amazonaws.com \"default\" not found"```

**Comment by njuettner** [comment]
`Containerd` is kind of working, I created PR https://github.com/giantswarm/aws-operator/pull/3370 to use an EBS volume for the `/var/lib/containerd` as the root disk similar to what we have for `docker`.

I noticed that in the beginning `kubelet` reports a warning about invalid capacity
```
  Warning  InvalidDiskCapacity      44m                kubelet                invalid capacity 0 on image filesystem
```

But this seems harmless from looking into upstream, see https://github.com/k3s-io/k3s/issues/1857. 
It only happens when bootstrapping a new node and the node become ready without problems.
